apps/kg/eval.py

```diff
@@ -99,17 +101,20 @@ def main(args):
     args.neg_sample_size_test = args.neg_sample_size
     if args.neg_sample_size < 0:
         args.neg_sample_size_test = args.neg_sample_size = eval_dataset.g.number_of_nodes()
+    args.eval_filter = not args.no_eval_filter
```

Comment:
I think we cat directly change self.add_argument('--no_eval_filter' to --eval_filter

Reply:
actually, i want to have filtering by default and disable filtering only when a user explicitly asks for it.

Reply:
We can use: store_false

Reply:
I see. This make sense